core/sail/memory/src/test/java/org/eclipse/rdf4j/sail/memory/QueryPlanRetrievalTest.java

```diff
@@ -30,6 +30,7 @@
 import org.eclipse.rdf4j.repository.sail.SailRepository;
 import org.eclipse.rdf4j.repository.sail.SailRepositoryConnection;
 import org.eclipse.rdf4j.rio.RDFFormat;
+import org.hamcrest.CoreMatchers;
```

Comment:
Can you replace with using assertJ matchers?

I'm actually kind of surprised this compiles - we switched from hamcrest to assertj a while back, and I thought we would have removed the hamcrest libs from the build path.

Reply:
Ah, of course, I keep forgetting that hamcrest core is actually a dependency of Junit 4... Still, I'd prefer if we stuck to assertJ instead of hamcrest, as a matter of consistency.

Reply:
I've fixed this now. Maybe we should exclude hamcrest in our pom files?